Makefile

```diff
@@ -136,6 +136,13 @@ verify-goimport:
 verify-yamllint:
 	yamllint .
 
+.PHONY: fix-yamllint
+fix-yamllint:
+ifeq (, $(shell which yamlfmt))
+	$(shell go install github.com/google/yamlfmt/cmd/yamlfmt@latest)
```

Comment:
maybe we should pin the version in the tools/mod.

Reply:
good catch! thanks

Reply:
Addressed